test_scripts/API/VehicleData/StabilityControlsStatus/008_GetVehicleData_StabilityControlsStatusNotAllowedByPolicy.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check processing of disallowed by policy stabilityControlsStatus data
+-- via GetVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed, stabilityControlsStatus NOT allowed by Policies
```

Comment:
@ydementieiev `stabilityControlsStatus param is NOT allowed by Policies`

Reply:
@AByzhynar [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/14/commits/1083dd2dbfb083e5b89ac7e7cf8bd751979f060a)